products/samsungmobile.md

```diff
@@ -19,6 +19,12 @@ eolColumn: Security Updates
 # 2. search on Google with the query : "<model_number> site:doc.samsungmobile.com",
 # 3. choose a page (preferably in english).
 releases:
+-   releaseCycle: "Galaxy S23 FE"
+    releaseDate: 2023-10-05
+    support: true
+    eol: 2028-10-05 # "five years of security updates" (https://security.samsungmobile.com/securityPost.smsb)
```

Comment:
Missed it on #3804, but S23 models are not listed on https://security.samsungmobile.com/securityPost.smsb.

Reply:
The S23 family is covered under "upcoming S series devices". The intent of that page was to establish their new 5-year security update policy going forward, which was also retroactively applied to some earlier S series devices (S21, S22).

Reply:
Sorry, should have read it with more attention.